### ISSUE TITLE
Official F-Droid Release

### ISSUE THREAD

**Issue by realmain** [Ethereal-Developers-Inc/OpenScan#27]
I see that the app has been released on Izzy's repo in issue #9, but some people do not want to add another repo to F-Droid.

It would be nice if it can be added to the official F-Droid repo.

[This is the inclusion policy](https://f-droid.org/en/docs/Inclusion_Policy/) that must be met to be able to submit. More info about submitting can be [found here](https://f-droid.org/en/contribute/).

[Here is the issue tracker](https://gitlab.com/fdroid/rfp/-/issues) to request your package to be available on F-Droid. After submitting, they can assist on what is needed to be done (if anything) to get it on F-Droid.

**Comment by veejayts** [comment]
Will do. Thanks for the links!

**Comment by veejayts** [comment]
This will take a a bit of time to get done. Facing issues trying to work with fdroid's app inclusion tools.

**Comment by veejayts** [comment]
Refer [MR on fdroid](https://gitlab.com/fdroid/fdroiddata/-/merge_requests/8336)

**Comment by opusforlife2** [comment]
PSA (because lots of activity has happened over at the fdroid repo which is not reflected here): The F-Droid inclusion process is on hold until a certain dependency is removed, making the inclusion process easier and less of a maintenance burden for everyone involved.

**Comment by opusforlife2** [comment]
@veejayts What's the latest update on this? Were you able to move away from that dependency?

**Comment by veejayts** [comment]
Still being worked on over at the [dev_v3](https://github.com/Ethereal-Developers-Inc/OpenScan/tree/dev_v3) branch.

**Comment by opusforlife2** [comment]
@veejayts Thanks!

Curiosity piqued. What made you decide on the OpenCV 3.x series instead of 4.x?

**Comment by jcperil** [comment]
Any update on this?

**Comment by licaon-kter** [comment]
https://gitlab.com/fdroid/rfp/-/issues/2336

**Comment by k4r4b3y** [comment]
Any more updates? I can't find the app on fdroid.

**Comment by licaon-kter** [comment]
last update is from 2021 https://gitlab.com/fdroid/fdroiddata/-/merge_requests/8336#note_505861119

**Comment by ukrbugreporter** [comment]
+1

**Comment by exaCORE42** [comment]
Are there any plans to add this application to the official F-Droid repository?

**Comment by k4r4b3y** [comment]
Well, being delisted from google play store doesn't have much relation to NOT being listed on fdroid. Am I wrong?

**Comment by exaCORE42** [comment]
The PSA says that OpenScan isn't being actively developed

**Comment by k4r4b3y** [comment]
Yeah that sucks. It is a pain in the ass to find a free software for camera-to-pdf doc scanners.

**Comment by veejayts** [comment]
@opusforlife2 v3 is a rework of the app for removing the dependency and fixing the bugs. No promises, currently I'm the only one working on this app, I'll make improvements, fixes and releases when I find the time.

FYI The dependency itself is not closed source, it's a binary build dependency which is not hosted on any repo (afaik) so it has been bundled directly.

Why can't it be unbundled easily? It's the core dependency of the app and an Android specific library (https://github.com/jhansireddy/AndroidScannerDemo). I'm re-implementing this core functionality in Flutter instead of tweaking the existing library in Java and compiling this to fit our app. Hosting it is no longer viable either since it is kinda old at this point and will require a rework in of itself.

Aside from this, Android has had a lot of updates regarding app permissions which I'm honestly yet to fully look into.

**Comment by Atemu** [comment]
Hi, why was this closed? Is the app on F-droid now?

**Comment by ghost** [comment]
I closed it to clean up my opened tickets

This app hasn't had a release since 2021

The dev can reopen issue if they are still interested
